Extensions/blacklist.js

```diff
@@ -670,13 +678,14 @@ XKit.extensions.blacklist = new Object({
 	hide_post: function(obj, word) {
 
 		if (XKit.extensions.blacklist.preferences.dont_block_me.value === true) {
-			if ($(obj).hasClass("is_mine") === true) {
+			if ($(obj).hasClass("is_mine") === true || $(obj).find(".mh_post_foot_control.icon-trash").length > 0) {
```

Comment:
does mobile not have any of the same data attr stuff? I feel like there's a more self-documenting selector we could be using here